PIL/XVThumbImagePlugin.py

```diff
@@ -58,14 +58,14 @@ def _open(self):
             s = self.fp.readline()
             if not s:
                 raise SyntaxError("Unexpected EOF reading XV thumbnail file")
-            if s[0] != b'#':
+            if i8(s[0]) != 35:  # ie. when not a comment: '#'
```

Comment:
Yeah, it's not great. What would be the way to do it?
Python 2: `s[0] == "#"`
Python 3: `s[0] == 35`